LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -950,8 +950,6 @@ $(function() {
 
     	// create save link once
     	if(!extendedToolbar) {
-    	    //adding the below line to overwrite the grid's export button instead of changing it in the grid file, since I'm not sure it's not used for anything else
```

Comment:
Was the export button not used for anything else?

Reply:
Export button is in formGrid.js, we added it here for the reasons listed in the comment but it caused errors since the functionality of the button was in formGrid.js, so we just changed it there and it works everywhere now